pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/session/PillarboxMediaController.kt

```diff
@@ -169,7 +171,7 @@ open class PillarboxMediaController internal constructor() : PillarboxPlayer {
     private lateinit var mediaController: MediaController
     private lateinit var playerSessionState: PlayerSessionState
 
-    private val listeners = mutableSetOf<PillarboxPlayer.Listener>()
+    private lateinit var listeners: ListenerSet<PillarboxPlayer.Listener>
```

Comment:
I think we should support a `null`-able `ListenerSet` here. This way we can release the previous one in `setMediaController()` (see my next comment).
```suggestion
    private var listeners: ListenerSet<PillarboxPlayer.Listener>? = null
```